src/constants.commands.ts

```diff
@@ -171,6 +171,7 @@ export const enum Commands {
 	ShowCommitsView = 'gitlens.showCommitsView',
 	ShowContributorsView = 'gitlens.showContributorsView',
 	ShowDraftsView = 'gitlens.showDraftsView',
+	FocusDraftsView = 'gitlens.views.drafts.focus',
```

Comment:
this command is no longer used in the walkthrough

Reply:
removed